persia/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.dev13"
\ No newline at end of file
+__version__ = "0.1.dev13"
```

Comment:
don't commit this file

this file is just a placeholder, and will be generated by pyproject.toml

Reply:
black formatter add blank newline 🤣